src/app/style/abstract/_icons.scss

```diff
@@ -162,24 +162,24 @@
     }
 
     &::before {
-        width: 7px;
-        height: 7px;
-        border: 1px solid var(--header-color);
+        width: calc(8px - var(--header-icon-stroke-width) * 2);
+        height: calc(8px - var(--header-icon-stroke-width) * 2);
+        border: var(--header-icon-stroke-width) solid var(--header-color);
         border-radius: 50%;
         top: 9px;
         right: 0;
         left: 0;
         margin: auto;
         box-shadow:
-            0 14px 0 3px var(--header-background),
-            0 14px 0 4px var(--header-color);
+            0 calc(13px + var(--header-icon-stroke-width)) 0 3px var(--header-background),
+            0 calc(13px + var(--header-icon-stroke-width)) 0 calc(var(--header-icon-stroke-width) + 3px) var(--header-color);
     }
 
     &::after {
         background-color: var(--header-background);
         bottom: -1px;
         left: 0;
-        height: 10px;
+        height: calc(11px - var(--header-icon-stroke-width));;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4874123)